linavigation-base/src/main/java/com/mapbox/navigation/logger/LoggerObserver.kt

```diff
@@ -0,0 +1,5 @@
+package com.mapbox.navigation.logger
+
+interface LoggerObserver {
```

Comment:
Since `setObserver` is not a part of the `Logger` interface, this interface can be shipped in the `liblogger` module instead.